core/src/main/java/org/elasticsearch/search/aggregations/bucket/terms/support/IncludeExclude.java

```diff
@@ -250,13 +304,30 @@ public IncludeExclude(long[] includeValues, long[] excludeValues) {
         this(convertToBytesRefSet(includeValues), convertToBytesRefSet(excludeValues));
     }
 
+    public IncludeExclude(int partition, int numPartitions) {
+        if (partition < 0 || partition >= numPartitions) {
+            throw new IllegalArgumentException(" Parition must be >=0 and < numPartition which is "+numPartitions);
```

Comment:
s/Parition/Partition/
Also the leading space in the error message looks unnecessary?